ports/hlslpp/hlslpp-config.cmake

```
@@ -0,0 +1,5 @@
+# Set the version number
+set(hlslpp_VERSION 3.5)

```

```suggestion
set(hlslpp_VERSION "@VERSION@")
```
and install with `configure_file ... @ONLY`. So it doesn't need updating again.